src/components/Popover/index.js

```diff
@@ -11,23 +11,40 @@ import PopoverWithoutOverlay from '../PopoverWithoutOverlay';
  * On small screen widths, it uses BottomDocked modal type, and a Popover type on wide screen widths.
  */
 function Popover(props) {
-    if (!props.fullscreen && !props.isSmallScreenWidth) {
+    const {isVisible, onClose, isSmallScreenWidth, fullscreen, animationInTiming, onLayout, animationOutTiming, disableAnimation, withoutOverlay, anchorPosition} = props;
```

Comment:
Just to confirm I understand why you did this, since we need `onClose` and `isVisible` as dependencies, we might as well split out all of them?

Reply:
yep! that makes sense.